libs/accounts/rate-limit/src/lib/models.ts

```diff
@@ -3,23 +3,33 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 /** The attributes we can count and block on. */
-export type BlockOn = 'ip' | 'email' | 'uid' | 'ip_email' | 'ip_uid';
+export const BLOCK_ON_ATTRIBUTES = [
+  'ip',
+  'email',
+  'uid',
+  'ip_email',
+  'ip_uid',
+] as const;
+export type BlockOn = (typeof BLOCK_ON_ATTRIBUTES)[number];
```

Comment:
There's not enough states here to merit this sorta thing. Please roll back these kind of changes across the board.